include/arch/riscv64/stdint.h

```diff
@@ -146,5 +146,19 @@ typedef uint64_t uintmax_t;
 #define _PRI_16 ""
 #define _PRI_32 ""
 #define _PRI_64 "l"
+#define _SCN_8   "hh"
+#define _SCN_16  "h"
+#define _SCN_32  ""
+#define _SCN_64  "ll"
```

Comment:
We use (unsigned) long for (u)int64_t type for this platform so you should use l prefix. Also compatible with _PRI_64

Reply:
That's right, sorry.